packages/edge/src/vercel-edge/index.ts

```diff
@@ -115,44 +101,26 @@ export function withEdgeMiddlewareAuth(
 
     const getToken = (options: ServerGetTokenOptions = {}) => {
       if (options.template) {
-        throw new Error(
-          'Retrieving a JWT template during edge runtime will be supported soon.',
-        );
+        throw new Error('Retrieving a JWT template during edge runtime will be supported soon.');
```

Comment:
Should this still be throwing? I think this PR has other code to retrieve on edge

Reply:
Looks like getAuthData is not hooked up to anything in `clerk/edge`. We will restructure `withEdgeMiddlewareAuth` a bit and make it follow the structure of the  `remix`/`nextjs` packages - this will be tackled in a separate PR